src/App.jsx

```diff
@@ -14,33 +17,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [visibleGoods, setVisibleGoods] = useState(goodsFromServer);
+  const [activeButton, setActiveButton] = useState(null);
+  const [isReversed, setIsReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  const sortAlph = () => {
+    const sorted = [...visibleGoods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    if (isReversed) {
+      sorted.sort().reverse();
+    } else {
+      sorted.sort();
```

Comment:
The sorting logic in `sortAlph` is slightly incorrect. The `reverse()` method should be applied after the `sort()` method, not chained directly. Consider separating the sorting and reversing into two steps to ensure clarity and correctness.